ntp-udp/src/raw_socket.rs

```diff
@@ -253,14 +251,14 @@ mod recv_message {
         };
 
         if mhdr.msg_flags & libc::MSG_TRUNC > 0 {
-            warn!(
+            tracing::info!(
                 max_len = packet_buf.len(),
                 "truncated packet because it was larger than expected",
             );
         }
 
         if mhdr.msg_flags & libc::MSG_CTRUNC > 0 {
-            warn!("truncated control messages");
+            tracing::info!("truncated control messages");
         }
```

Comment:
This very likely is a bug on our end, so should be warn in my view (also not remotely triggerable unless I misunderstand control messages)